SYSTEM_TESTS.md

```diff
@@ -0,0 +1,349 @@
+ 
+
+# Guide on System Tests in Ruby
+
+System tests in Ruby are special types of tests.
+
+They allow us to interact with our web app, test the UX and JavaScript of our application.
+
+This repository uses [Capybara](https://github.com/teamcapybara/capybara) library to perform system tests. It simulates user's interaction with our web app (visiting pages, clicking buttons, filling forms...)
+
+System tests are the slowest type of tests and we need to be careful when writing them.
```

Comment:
Could you add detail here to explain that they need to be minimal and that because of their slow speed, every test need to be crucial and efficient? Thanks!

Reply:
The sentence on line 11 says exactly that.

Reply:
No I know but in just says `we need to be careful` and I think it would be nice to detail that that means the tests need to be efficient and minimal

Reply:
Well, I've covered some of the edge cases of system tests. It really depends on what someones mean by minimal. In the context of tests in general, I don't think that 'minimal' is the right word to use. A person writing tests should know what code needs to be tested.

In the end, @jywarren decides if a test should be added or not. In other words, is the test crucial to certain functionality?